install/kubernetes/helm/istio/values.yaml

```diff
@@ -8,6 +8,19 @@ global:
   # Default tag for Istio images.
   tag: release-1.0-latest-daily
 
+  # Gateway used for legacy k8s Ingress resources. By default it is
+  # using 'istio:ingress', to match 0.8 config. It requires that
+  # ingress.enabled is set to true. You can also set it
+  # to ingressgateway, or any other gateway you define in the 'gateway'
+  # section.
+  ingressSelector: ingress
```

Comment:
Should this to be set to the ingress gateway instead as ingress.enabled is set to false by default now in 1.0 IIUC?   Or are you thinking this won't take any effect by default since ingress.enabled is set to false?

Reply:
Not sure. I was thinking default should be to use gateways and no ingress.
If users has or want Ingress - safest option is to ingress.enabled=true.

If we set it by default to ingressgateway - it's a bit riskier, since same domain as both ingress and gateway will result in errors. But it is still a valuable option for some users.

Reply:
I see, this only applies to istio-autogenerated-k8s-ingress so setting to ingress by default does make sense now.  Thanks!